src/components/dependencies/BDBARescoring.js

```diff
@@ -1829,6 +1968,68 @@ Rescore.propTypes = {
 }
 
 
+const sprintNameForRescoring = (rescoring) => {
+  const now = new Date(new Date().setHours(0, 0, 0, 0))
+
+  const findingIsResolved = () => {
+    if (rescoring.applicable_rescorings.length === 0) return false
+    const rescoringsOrderedBySpecificity = orderRescoringsBySpecificity(rescoring.applicable_rescorings)
+    return rescoringsOrderedBySpecificity[0].data.severity === SEVERITIES.NONE
+  }
+
+  if (findingIsResolved())
+    return 'Resolved'
```

Comment:
Also introduce an enum/frozen-object for sprint "meta names"

Reply:
This util function is not specific for BDBA, right?
If so: please move to more generic module